src/Main.java

```diff
@@ -5,8 +5,36 @@
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code scrijven voor de hoofd-opdracht
+        Integer [] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String [] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
 
+        Translator translator = new Translator(alphabetic, numeric);
+
+        boolean play = true;
+        String ongeldig = "Ongeldige invoer";
+        Scanner scanner = new Scanner(System.in);
+
+        while (play) {
+            System.out.println("Type 'x' om te stoppen \n Type 'v' om te vertalen");
+            String input = scanner.nextLine();
+            if (Objects.equals(input, "x")) {
```

Comment:
In plaats van `(Objects.equals(input, "x"))` had je ook `(input.equals("x"))` kunnen gebruiken.